R/pkg_ref_class_coersion.R

```diff
@@ -3,7 +3,7 @@
 #' @export
 as_tibble.pkg_ref <- function(x, ...) {
   as_tibble(vctrs::new_list_of(list(x),
-    ptype = pkg_ref(),
+    ptype = list(),
```

Comment:
`ptype` needs to be a vector type indicating what type of vector-ish object this should be, not a prototype element. This is enforced in the upcoming `vctrs` release and needs to be fixed before early january.